airflow/models/baseoperator.py

```diff
@@ -804,12 +804,15 @@ def render_template(      # pylint: disable=too-many-return-statements
         if not jinja_env:
             jinja_env = self.get_template_env()
 
+        from airflow.models.xcom_arg import XComArg
```

Comment:
Is it defined here to avoid circular dep?

Reply:
Exactly so